spec.md

```diff
@@ -2888,3 +2888,32 @@ services:
   foo:
     build: null
 ```
+## Dependency on other compose projects
+
+A Compose project can declare dependency on another compose project, managed by another team
+or shared with others. This allows to keep a compse file reasonably complicated for the limited
+amount of resources a single team has to declare for it's own sub-domain within a larger deployment.
+
+`require` top-level section is used to define dependency to another compose application (or subdomain).
+Each path listed in this section MUST be loaded as an individual compose model with it's own project
```

Comment:
are you also imagining that each individual compose model will have it's own .env file and do it's own env resolution?

Reply:
good point, didn't considered this (which is out of the compose spec scope)

I tried to make this minimalist on first run, but we obviously could have a "long syntax" to offer fine grain control on resources we import into current model:
```yaml
include:
   - source: ../another/compose.yaml
     project-directory: /tmp
     excludes: ["db"]
     rename: foo=bar
     env: ../another/.env
```

Reply:
:+1: ya, i appreciate the update. just make it an error if the names clash for now, and we can iterate later.